docs/fundamentals/executor/hub/push-executor.md

```diff
@@ -67,3 +67,12 @@ If you want to create a new tag for an existing Executor, you can also add the `
 ```bash
 jina hub push [--public/--private] --force-update <NAME> --secret <SECRET> -t TAG <path_to_executor_folder>
 ```
+
+### Immutable tags
```

Comment:
Can we have a description of why is this desirable? I cannot see the point of the feature with this description

Reply:
@nomagick Please click the "Resolve conversation" if you think this comment has been addressed.